src/server/modules/api/external-v1/ApiController.ts

```diff
@@ -0,0 +1,74 @@
+import Express from 'express'
+import { inject, injectable } from 'inversify'
+import Sequelize from 'sequelize'
+
+import { logger } from '../../../config'
+import { DependencyIds } from '../../../constants'
+import jsonMessage from '../../../util/json'
+import { AlreadyExistsError, NotFoundError } from '../../../util/error'
+
+import { UrlManagementService } from '../../user/interfaces'
+import { MessageType } from '../../../../shared/util/messages'
+import { StorableUrlSource } from '../../../repositories/enums'
+import { StorableUrl } from '../../../repositories/types'
+
+import { ApiUrl, UrlCreationRequest } from '.'
+
+@injectable()
+export class ApiController {
```

Comment:
I think we can name this ApiV1Controller

Reply:
done in [ad18d7c](https://github.com/opengovsg/GoGovSG/pull/2069/commits/ad18d7c82fee948a7aafe8a1107a36870589304f)